docs/docsite/rst/user_guide/playbooks_reuse_roles.rst

```diff
@@ -78,9 +78,11 @@ Roles may also include modules and other plugin types in a directory called ``li
 Storing and finding roles
 =========================
 
-By default, Ansible looks for roles in two locations:
+By default, Ansible looks for roles in four locations:
 
 - in a directory called ``roles/``, relative to the playbook file
+- in ``~/.ansible/roles``
+- in ``/usr/share/ansible/roles``
 - in ``/etc/ansible/roles``
 
```

Comment:
same with this one as above, should not be listed (unless we want to show 'defaults to' and put all the options there for the variable).